php/blocks/controls/class-repeater.php

```diff
@@ -24,19 +24,28 @@ class Repeater extends Control_Abstract {
 	/**
 	 * Field variable type.
 	 *
-	 * The Repeater control is an array of objects, with each row being an object.
-	 * For example, a repeater with one row might be [ { 'example-text': 'Foo', 'example-image': 4232 } ].
+	 * The Repeater control is an array of arrays, with each row being its own array.
+	 * For example, a repeater with two rows might be:
+	 * [ 'rows': [ 0: [ 'example-text': 'Foo', 'example-image': 42 ], 1: [ 'example-text': 'Bar', 'example-image': 32 ] ] ].
 	 *
 	 * @var string
 	 */
-	public $type = 'object';
+	public $type = 'array';
```

Comment:
Good thing to bring up. In PHP, this would be an `array()`, and in JS it'd be an object. I don't see many examples of array or object attributes in Core blocks.

Reply:
Ah – makes much more sense for `$type` to represent the Javascript type, rather than the PHP type. Addressed in https://github.com/getblocklab/block-lab/pull/429/commits/2b61b6476075fa1c01ec3ae866065fe1ba33fc7a.